desktop_version/src/Game.cpp

```diff
@@ -336,7 +336,8 @@ void Game::init(void)
     stat_trinkets = 0;
 
     state = 1;
-    statedelay = 0;
+    setstatedelay(0);
```

Comment:
Overzealous find-and-replace? This should stay as `statedelay = 0;` because it's an init function.